packages/xo-vmdk-to-vhd/src/vmdk-read-table.js

```diff
@@ -54,10 +54,9 @@ export default async function readVmdkGrainTable(fileAccessor) {
   const grainSize =
     getLongLong(headerBuffer, GRAIN_SIZE_OFFSET, 'grain size') * SECTOR_SIZE
   const grainCount = Math.ceil(capacity / grainSize)
-  const numGTEsPerGT = getLongLong(
-    headerBuffer,
+  const numGTEsPerGT = new DataView(headerBuffer).getUint32(
     NUM_GTE_PER_GT_OFFSET,
-    'num GTE per GT'
+    true
```

Comment:
Can you explain this change?

What I understand is that you used to parse it as a Uint64 but now you changed it to Uint32.

Reply:
yeah, I was wrong before:
<img width="384" alt="capture d ecran 2018-11-28 a 11 33 41" src="https://user-images.githubusercontent.com/46941/49173660-8b286f80-f301-11e8-8c06-46656a6d2e75.png">